rust/sealevel/programs/hyperlane-sealevel-token/tests/functional.rs

```diff
@@ -173,6 +194,17 @@ async fn initialize_hyperlane_token(
     );
     banks_client.process_transaction(transaction).await?;
 
+    // Set destination gas configs
+    set_destination_gas_config(
```

Comment:
There's a lot of code duplication in the three warp route functional tests, we should refactor this in another PR

Reply:
haha yes this was pretty egregious of me tbh - time constraints led me down this path but started to feel the pain of the code duplication in the PR for sure. agree on the refactor in the future